src/askvm/resources/core/__test__/let.test.ts

```diff
@@ -16,27 +16,9 @@ function ask(code: string) {
 }
 
 describe(`let`, function () {
-  it(`should throw when trying to assign to const`, async function () {
+  it(`should assign value to a let variable on initialization`, async function () {
     const value = 6;
-    await expect(
-      ask(`ask(const('a',${value}),call(get('get'),assign('a',4),'a'))`)
-    ).rejects.toThrow(`Cannot assign to a constant variable "a"`);
-  });
-
-  it(`should assign value on initialization`, async function () {
-    const value = 6;
-    await expect(
-      ask(`ask(let('a',${value}),call(get('get'), 'a'))`)
-    ).resolves.toEqual(value);
-  });
-
-  it(`should assign value with the assign resource`, async function () {
-    const initialValue = 6;
-    const assignedValue = 4;
-    const resolvedValue = await ask(
-      `ask(let('a',${initialValue}),assign('a',${assignedValue}))`
-    );
-    expect(resolvedValue).toEqual(assignedValue);
+    await expect(ask(`ask(let('a',${value}))`)).resolves.toEqual(value);
```

Comment:
This only checks if `let` returns the assigned value. Can we add checking if the value was actually set?

Reply:
Yes, changed the ask to``ask(let('cocoCount',${value}),get('cocoCount'))``